chain/params.go

```diff
@@ -1,18 +1,54 @@
 package chain
 
 import (
+	"errors"
 	"math/big"
+	"sort"
+	"strconv"
 
 	"github.com/0xPolygon/polygon-edge/types"
 )
 
+var (
+	// ErrBurntContractAddressMissing is the error when a contract address is not provided
+	ErrBurntContractAddressMissing = errors.New("burnt contract address missing")
+)
+
 // Params are all the set of params for the chain
 type Params struct {
 	Forks          *Forks                 `json:"forks"`
 	ChainID        int                    `json:"chainID"`
 	Engine         map[string]interface{} `json:"engine"`
 	Whitelists     *Whitelists            `json:"whitelists,omitempty"`
 	BlockGasTarget uint64                 `json:"blockGasTarget"`
+
+	// Governance contract where the token will be sent to and burnt in london fork
+	BurntContract map[string]string `json:"burntContract"`
+}
+
+// CalculateBurntContract calculates burn contract address for the given block number
+func (p *Params) CalculateBurntContract(number uint64) (types.Address, error) {
```

Comment:
If this is block number, shouldn't it be `block` or `height`?

Reply:
Renamed